Commons/src/main/java/com/yukthi/webutils/common/IWebUtilsCommonConstants.java

```diff
@@ -118,4 +118,8 @@
 	 */
 	public String PATTERN_EMAIL = "[\\w\\.\\-]+\\@[\\w\\.\\-]+\\.[\\w]+";
 	
+	/** 
+	  * The pattern url link. 
+	  **/
+	 public String PATTERN_URL_LINK = "(https?://.+)?";

```

Comment:
Your pattern should be https?\\:\\/\\/.+     
In patterns special characters should be always escaped using \
And also better dont hard code protocol.. Make it \\w+\\:\\/\\/.+